lib/surrealist/class_methods.rb

```diff
@@ -55,6 +55,26 @@ def json_schema(&_block)
       SchemaDefiner.call(self, yield)
     end
 
+    # A DSL method to return the defined schema.
+    # @example DSL usage example
+    #   class Person
+    #     include Surrealist
+    #
+    #     json_schema do
+    #       { name: String }
+    #     end
+    #
+    #     def name
+    #       'Parent'
+    #     end
+    #   end
+    #
+    #   Person.defined_schema
+    #   # => { name: String }
+    def defined_schema
+      instance_variable_get('@__surrealist_schema')
```

Comment:
Should we raise `UnknownSchemaError` or any other error if any schema has been defined?

Reply:
Yes, I think we should